buildlib/az-helpers.sh

```diff
@@ -196,7 +196,7 @@ check_release_build() {
             [[ "$title" == "${title_mask}"* ]] && launch=True;
         done
     fi
-
+    set +x
```

Comment:
maybe use azure_set_variable

Reply:
hmm seems release pipeline is not run on this pr
maybe need to add isOutput=true to azure_set_variable

Reply:
Right. It makes a var globally available, so it could be used between stages.